Source/WebKit/WebProcess/WebPage/IPCTestingAPI.cpp

```diff
@@ -1770,6 +1864,7 @@ void JSIPC::finalize(JSObjectRef object)
 const JSStaticFunction* JSIPC::staticFunctions()
 {
     static const JSStaticFunction functions[] = {
+        { "getConnectionForProcessTarget", getConnectionForProcessTarget, kJSPropertyAttributeDontDelete | kJSPropertyAttributeReadOnly },
```

Comment:
We typically don't prefix a method with "get" prefix unless it has an out argument.
So I think we should call this `connectionForProcessTarget` instead.